src/containers/Dashboard.vue

```diff
@@ -97,27 +96,15 @@ import TourWindow from '@/components/Modals/TourWindow.vue';
 
 import surfing from '@/assets/images/surfing.svg';
 
-import {
-  FirestoreUserName,
-  FirestoreSemester,
-  AppOnboardingData,
-  AppBottomBarCourse,
-  FirestoreSemesterCourse,
-} from '@/user-data';
 import computeRequirements from '@/requirements/reqs-functions';
 import { CourseTaken, SingleMenuRequirement } from '@/requirements/types';
 import getCourseEquivalentsFromUserExams from '@/requirements/data/exams/ExamCredit';
 import store, { initializeFirestoreListeners, subscribeRequirementDependencyChange } from '@/store';
 import { semestersCollection } from '@/firebaseConfig';
 import getCurrentSeason, { getCurrentYear } from '@/utilities';
 
-Vue.component('course', Course);
 Vue.component('semesterview', SemesterView);
 Vue.component('requirements', Requirements);
```

Comment:
Are we waiting on handling these until after the migration to the global data store?

Reply:
Yes